MARIE_AND_BERT/Marie/EntityLinking/readme.md

```diff
@@ -0,0 +1,4 @@
+The BLINK library is included in the source code of this project due to changes on the 
```

Comment:
Now that you have removed the clone instruction from one of the other readme files, you may want to add this link `https://github.com/facebookresearch/BLINK` is where you originally got it from.